### PR TITLE
fix bootstrapann input

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] fix bootstrapann input
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+- Input to BootsrapAnn is now supplied in a single channel #151(https://github.com/genomic-medicine-sweden/tomte/pull/151)
```
Comment (Lucpen) [inline CHANGELOG.md:12]:
```suggestion
- Input to BootsrapAnn is now supplied in a single channel #[151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+- Input to BootsrapAnn is now supplied in a single channel #[151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```
Comment (Lucpen) [inline CHANGELOG.md:12]:
```suggestion
- Input to BootsrapAnn is now supplied in a single channel [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+- Input to BootsrapAnn is now supplied in a single channel [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```
Comment (fellen31) [inline CHANGELOG.md:12]: If this caused sample mixups in previous versions of the pipeline (?), could you add this to the text?

Comment (Lucpen) [reply CHANGELOG.md:12]: Good point!

Comment (Lucpen) [inline CHANGELOG.md:12]:
```suggestion
- Input to BootsrapAnn is now supplied in a single channel. Previously they were supplied in separate channels, which could cause mix-ups if more than one sample was supplied [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+- Input to BootsrapAnn is now supplied in a single channel. Previously they were supplied in separate channels, which could cause mix-ups if more than one sample was supplied [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```
Comment (fellen31) [inline CHANGELOG.md:12]:
```suggestion
- Input to BootstrapAnn is now supplied in a single channel. Previously they were supplied in separate channels, which could cause mix-ups if more than one sample was supplied [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```

**Review by fellen31 — APPROVED** [review]
Nice!

**Pull request merged**